src/python/fikriazain-name-shorten.py

```diff
@@ -0,0 +1,18 @@
+#  Shorten a name
+#  For example Michael Henderson --> Henderson, M.
+#  Hope you enjoy
+
+def name_shortener(user_name):
```

Comment:
function name should the same as in the filename

Reply:
ohh i see okay sorry